### ISSUE TITLE
Occasional Girder crashes on nimbusimage.com

### ISSUE THREAD

**Issue by arjunrajlab** [Kitware/UPennContrast#666]
Here's an error I saw in the output from the girder log:

```
172.21.0.1 - - [24/Apr/2024:18:03:10] "GET /api/v1/dataset_view?limit=20&offset=0&sort=lastViewed&sortdir=-1 HTTP/1.1" 401 55 "https://nimbusimage.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/124.0.0.0 Safari/537.36"
(trapped) error reading bcrypt version
Traceback (most recent call last):
  File "/venv/lib/python3.11/site-packages/passlib/handlers/bcrypt.py", line 620, in _load_backend_mixin
    version = _bcrypt.__about__.__version__
              ^^^^^^^^^^^^^^^^^
AttributeError: module 'bcrypt' has no attribute '__about__'
```

I've been seeing this error every so often on nimbusimage.com. Not sure exactly how to reproduce it, but can try. Tends to happen when uploading files. The files aren't super large, around 6GB, so not sure what's up.

**Comment by arjunrajlab** [comment]
I wonder if this is related to https://github.com/Kitware/UPennContrast/issues/667, given that both problems cropped up around the same time.

**Comment by arjunrajlab** [comment]
I updated with the fix for #667 and this error still seems to happen…

**Comment by manthey** [comment]
This should be resolved by https://github.com/girder/girder/pull/3538
